controllers/add_services.go

```diff
@@ -89,7 +90,7 @@ func (a addServices) reconcile(ctx context.Context, r *FoundationDBClusterReconc
 					return &requeue{curError: err}
 				}
 			} else if k8serrors.IsNotFound(err) {
-				// Create a new service
+				log.Info("Creating service", "name", service.Name)
```

Comment:
That should be a debug statement not an info statement. I would prefer if we also log the corresponding namespace and cluster name.